lib/barcelona/plugins/secure_instance_plugin.rb

```diff
@@ -0,0 +1,80 @@
+# SecureInstance plugin
+# This plugin configure container instance and bastion instance more secure
+# Usage: bcn district put-plugin discrict1 secure_instance
+
+module Barcelona
+  module Plugins
+   
+    class SecureInstancePlugin < Base
+
+      def on_container_instance_user_data(_instance, user_data)
+        user_data.extend SecureUserData
+        user_data.install_tools_for_pcidss
+        user_data.install_tcp_wrappers
```

Comment:
can you explain why is `tcp_wrappers` needed?

Reply:
Because a CIS rule requires it.

https://github.com/degica/barcelona/blob/abf9936a42ad943005a3f276746029ed0aa17989/spec/lib/barcelona/plugins/secure_instance_plugin_spec.rb#L37

Reply:
I see, so we don't install it in the bastion server because Inspector didn't complain?

Reply:
Yes, tcp_wrappers is already installed on bastion.